packages/openapi-generator/src/parser/schema.ts

```diff
@@ -170,11 +170,15 @@ function parseEnumSchema(
   return {
     type,
     enum: (schema.enum || []).map(entry =>
-      type === 'string' ? `'${entry}'` : entry
+      type === 'string' ? toEnumString(entry) : entry
     )
   };
 }
 
+function toEnumString(input: string): string {
```

Comment:
Consider a function name with a verb, e. g. `getStringEnumValue` or similar. This is also in our [style guide](https://github.com/SAP/cloud-sdk-js/blob/main/STYLEGUIDE.md#use-verbs-in-function-names).

Reply:
change. Regarding this in the style guide. We use often noun like funciton like `tsConfig()` when we create objects in the generator. Should we add this to the style guide?